hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -293,6 +295,12 @@ public Operation convert(String value) throws ParameterException {
     @Parameter(names = {"--checkpoint"}, description = "Resume Delta Streamer from this checkpoint.")
     public String checkpoint = null;
 
+    @Parameter(names = {"--bootstrap-from"}, description = "Initial bootstrap from this path")
```

Comment:
this is a path where the current dataset resides? I can see that we can eventually use this for actually bootstrapping the dataset.. but wondering if for now, the CheckpointProvider just takes a property containing the base path for reading/.computing checkpoints? 

i.e we can remove `--bootstrap-from`.. it almost sounds like we are actually bootstrapping the data.

Reply:
yea, with the `props` file, we can get the path there. Don't need this field anymore